components/server/src/billing/entitlement-service-ubp.ts

```diff
@@ -70,7 +70,7 @@ export class EntitlementServiceUBP implements EntitlementService {
 
     protected async checkUsageLimitReached(
         user: User,
-        organizationId: string | undefined,
+        organizationId: string,
```

Comment:
out of scope, but we could start using `uuid` type here (and other places) for even more type safety